internal/controllers/synthesis/integration_test.go

```diff
@@ -238,3 +242,38 @@ func TestControllerSwitchingSynthesizers(t *testing.T) {
 		assert.NotEqual(t, comp.Status.CurrentSynthesis.ResourceSlices, initialSlices)
 	})
 }
+
+func TestControllerBackoff(t *testing.T) {
```

Comment:
Unrelated to the PR: moved from backoff_test.go since it was the only test in the file